<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(255, 255, 255);
font: 20pt &quot;Ubuntu&quot;;
background: rgb(0,0,0,50);
border-radius: 16px;
color: rgb(252, 233, 79);</string>
   </property>
   <property name="text">
    <string>开灯</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(255, 255, 255);
font: 20pt &quot;Ubuntu&quot;;
background: rgb(0,0,0,50);
border-radius: 16px;
color: rgb(115, 210, 22);</string>
   </property>
   <property name="text">
    <string>光源检测</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_result">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(255, 255, 255);
background: rgb(0,0,0,50);
border-radius: 16px;
color: rgb(115, 210, 22);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_connect">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(255, 255, 255);
font: 20pt &quot;Ubuntu&quot;;
background: rgb(0,0,0,50);
border-radius: 16px;
color: rgb(12, 250, 250);</string>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="result">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(155, 255, 255);
font: 20pt &quot;Ubuntu&quot;;
background: rgb(0,0,0,50);
border-radius: 16px;
color: rgb(115, 210, 22);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>349</width>
     <height>529</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/impa/1714960485247.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>lineEdit_result</zorder>
  <zorder>pushButton_connect</zorder>
  <zorder>result</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
